tests/async/conftest.py

```diff
@@ -38,8 +39,10 @@ def utils() -> Generator[Utils, None, None]:
 
 # Will mark all the tests as async
 def pytest_collection_modifyitems(items: List[pytest.Item]) -> None:
-    for item in items:
-        item.add_marker(pytest.mark.asyncio)
+    pytest_asyncio_tests = (item for item in items if is_async_test(item))
```

Comment:
This is token from [here](https://pytest-asyncio.readthedocs.io/en/latest/how-to-guides/run_session_tests_in_same_loop.html).